src/components/miscellaneous/AspectRatioImg.tsx

```diff
@@ -46,27 +46,7 @@ const AspectRatioPolyfill = styled('div', { shouldForwardProp: (prop) => prop !=
 
 const AspectRatioImg = ({ alt, borderRadius, className, ratio = 21 / 9, src, sx }: AspectRatioImgProps) => {
   const [imgUrl, setImgUrl] = useState(src || TIHLDELOGO);
-  useEffect(() => {
-    setImgUrl(src || TIHLDELOGO);
-  }, [src]);
```

Comment:
useEffect er nødvendig for å oppdatere `imgUrl` hvis/når src endres

Reply:
Src kan kun endres dersom props til componenten endres. Dersom src (props) endres vil componenten re-rendere uansett. Fra React docs : "A React component re-renders when its state or props change.". Dermed blir useEffecten ubrukelig i dette tilfellet.

Reply:
Det er feil. Riktig at komponenten re-rendres når props endres, men useState beholdes. Det er hele poenget med useState. Se https://stackoverflow.com/a/54866051

Reply:
Hei! Det er helt riktig, litt lenge siden jeg har brukt react. Bra du catcha det!